nodes/src/nodes/tool_vectordb/vectordb_driver.py

```
@@ -0,0 +1,389 @@
+                doc_filter.parent = parent
+
+        # Build the question for the store's search interface
+        question = QuestionText(
+            text=query_text,
+            topK=top_k,
+        )
+
+        # Use semantic search (the primary use case for vector DBs)
+        try:
+            docs: List[Doc] = self._store.searchSemantic(question, doc_filter)
+        except Exception as e:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)